File: mylibrary/src/main/java/net/dongliu/apk/parser/bean/GlEsVersion.java
package net.dongliu.apk.parser.bean;


import androidx.annotation.NonNull;

/**
 * the glEsVersion apk used.
 *
 * @author dongliu
 */
public class GlEsVersion {
    private final int major;
    private final int minor;
    private final boolean required;

    public GlEsVersion(final int major, final int minor, final boolean required) {
        this.major = major;
        this.minor = minor;
        this.required = required;
    }


    public int getMajor() {
        return this.major;
    }

    public int getMinor() {
        return this.minor;
    }

    public boolean isRequired() {
        return this.required;
    }

    @NonNull
    @Override
    public String toString() {
        return this.major + "." + this.minor;
    }

}

File: mylibrary/src/main/java/net/dongliu/apk/parser/bean/GlEsVersion.kt
package net.dongliu.apk.parser.bean

/**
 * the glEsVersion apk used.
 * 
 * @author dongliu
 */
class GlEsVersion(val major: Int, val minor: Int, val isRequired: Boolean) {
    override fun toString(): String {
        return this.major.toString() + "." + this.minor
    }
}
